src/webapp/components/dataset-wizard/IndicatorsDataSet.tsx

```diff
@@ -166,6 +156,11 @@ export const IndicatorsDataSet = React.memo((props: IndicatorsDataSetProps) => {
 
     const indicatorsPerType = useBuildTotalByKey(dataSet.indicators, indicator => indicator.type);
 
+    const toggleCompanionTable = React.useCallback(
+        () => companionTableActions.toggle(),
+        [companionTableActions]
+    );
+
```

Comment:
We can simply put `const toggleCompanionTable = companionTableActions.toggle`, right?